hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestKeyInputStream.java

```diff
@@ -64,25 +71,38 @@
   private static String volumeName;
   private static String bucketName;
   private static String keyString;
+  private static ChunkLayoutTestInfo chunkLayout;
+
+  @Parameterized.Parameters
+  public static Collection<Object[]> layouts() {
+    return Arrays.asList(new Object[][] {
+        {ChunkLayoutTestInfo.FILE_PER_CHUNK},
+        {ChunkLayoutTestInfo.FILE_PER_BLOCK}
+    });
+  }
 
+  public TestKeyInputStream(ChunkLayoutTestInfo layout) {
+    this.chunkLayout = layout;
+  }
   /**
    * Create a MiniDFSCluster for testing.
    * <p>
    * Ozone is made active by setting OZONE_ENABLED = true
    *
    * @throws IOException
    */
-  @BeforeClass
-  public static void init() throws Exception {
-    chunkSize = 100;
+  @Before
+  public void init() throws Exception {
+    chunkSize = 1024 * 1024 * 4;
```

Comment:
I will try to reduce the chunkSize. One limitation here is chunkSize cann't be less than the mininum bytes-per-checksume value which is 256KB, otherwise the TestKeyInputStream will always succeed event without this patch.